lib/charts/src/chart_series_impl.dart

```diff
@@ -43,10 +43,13 @@ class DefaultChartSeriesImpl extends ChangeNotifier implements ChartSeries {
 
   Iterable<int> get measures => _measures;
 
-  set measureAxisIds(Iterable<String> value) => _measureAxisIds = value;
```

Comment:
Yes. Because setters are supposed to have `void` type, if you use the '=>' you are returning a non-void value and the setter is not strong-mode compliant